packages/cozy-harvest-lib/src/helpers/queries.js

```diff
@@ -1,15 +1,49 @@
 import CozyClient, { Q } from 'cozy-client'
 
-const DEFAULT_CACHE_TIMEOUT_QUERIES = 9 * 60 * 1000
+const DEFAULT_CACHE_TIMEOUT_QUERIES = 10 * 60 * 1000
 const defaultFetchPolicy = CozyClient.fetchPolicies.olderThan(
   DEFAULT_CACHE_TIMEOUT_QUERIES
 )
 
-export const buildKonnectorQuery = slug => ({
-  definition: () =>
-    Q('io.cozy.konnectors').getById(`io.cozy.konnectors/${slug}`),
+export const buildKonnectorQuery = (slug, source) => {
+  const query = {
+    definition: () =>
+      Q('io.cozy.konnectors').getById(`io.cozy.konnectors/${slug}`),
+    options: {
+      as: `konnector-${slug}-${source}`,
+      fetchPolicy: defaultFetchPolicy
+    }
+  }
+  const queryDef = query.definition()
+  queryDef.sources = [source]
+
+  return { definition: queryDef, options: query.options }
+}
+
+export const buildIsAccountConnectedQuery = sourceAccount => ({
```

Comment:
This is not doing anything related to "connected". I propose to rename into:

```
const buildAccountQuery
```

This way, it can be reused in other contexts that for the "connected" purpose.